esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -71,6 +71,10 @@ int baudrate = 57600;
 // comment out for default setting
 #define WIFI_POWER  WIFI_POWER_2dBm // WIFI_POWER_MINUS_1dBm is the lowest possible, WIFI_POWER_19_5dBm is the max
 
+// WiFi channel
+// comment out for default setting
+int WIFI_CHANNEL = 13; // Use 13 (2461-2483 MHz) since it has least overlap with 2.4 freqs
```

Comment:
lower case lower case :D

Reply:
Thanks for the patience :) I see the pattern now:
- lower for variables
- upper for defines

Reply:
👍